controlpanel/frontend/jinja2/user-detail.html

```diff
@@ -13,6 +13,27 @@
 <span class="govuk-caption-xl">User</span>
 <h1 class="govuk-heading-xl">{{ page_title }}</h1>
 
+{% if unused %}
+<div class="govuk-grid-row">
+  <div class="govuk-column-two-thirds">
+
+    <div class="govuk-error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+
+      <h2 class="govuk-heading-m govuk-error-summary-heading">
+      This Account Could be Unused
+      </h2>
+
+      <p>
+        <strong>The user of this account has not logged in for over three
```

Comment:
Don't know whether we should be explicit and say "has not logged in the Control Panel for over three months" or not (to avoid any possible confusion)

I know it's a bit extreme but **potentially** a user could have not logged into the Control Panel for months but still actively working on things on GitHub.

Just a thought.

Reply:
Actually I see the code in list view checks for last login in Auth0 and then fall back to `last_login` in CP/DB record.

Reply:
:+1: